apps/connect/src/utils/sanctions.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  CHAIN_ID_ALGORAND,
+  CHAIN_ID_ARBITRUM,
+  CHAIN_ID_AVAX,
+  CHAIN_ID_BSC,
+  CHAIN_ID_BTC,
+  CHAIN_ID_CELO,
+  CHAIN_ID_OPTIMISM,
+  CHAIN_ID_POLYGON,
+  CHAIN_ID_SOLANA,
+  ChainId,
+  isEVMChain,
+} from "@certusone/wormhole-sdk";
+import { CLUSTER } from "../env/common";
+interface SanctionResponse {
+  accountExternalId: string;
+  address: string;
+  addressRiskIndicators: {
+    category: string;
+    categoryRiskScoreLevel: number;
+    riskType: "OWNERSHIP" | "COUNTERPARTY" | "INDIRECT";
+  }[];
+  entities: {
+    category: string;
+    riskScoreLevel: number;
+    entity: string;
+  }[];
+  trmAppUrl: string;
+}
+
+// TRM screening chain names map with wormhole chain ids
+// https://documentation.trmlabs.com/tag/Supported-Blockchain-List
+export const getTrmChainName = (chain: ChainId) => {
```

Comment:
this seems to be duplicated, is that okay? 

see: https://github.com/XLabs/portal-bridge-ui/pull/949/files#diff-3510a15bedcd814dcf39355e1a69bd9d8b39861f0c1481f1cb13c9ee853aaa81R33